envoy/allowed-services.template.lua

```diff
@@ -0,0 +1,147 @@
+-- IMPORTANT: All Services for the PATH Service Gateway must be listed here for Envoy Proxy to forward requests to PATH.
+--
+-- The service IDs configured here are used in the `envoy.filters.http.lua` HTTP filter defined in `.envoy.yaml` config file.
+-- The `.allowed-services.lua` file must be mounted as a file in the Envoy Proxy container at `/etc/envoy/.allowed-services.lua`.
+--
+-- If you wish to define aliases for existing services, you must define the alias as the key and the service ID as the value.
+--
+-- eg 1. the service ID ["F000"] = "F000" enables Envoy to forward requests with the subdomain "F000.path.grove.city" to PATH with the service ID "F000".
+-- eg 2. the alias ["pocket"] = "F000" enables Envoy to forward requests with the subdomain "pocket.path.grove.city" to PATH with the service ID "F000".
+--
+-- To utilize PATH's Quality of Service (QoS) features, the service ID must match the value in PATH's `qos` module.
+-- TODO_IMPROVE(@commoddity): Add link to the file & line in the QoS module once 'no-op' QoS feature is completed.
```

Comment:
Can we do this in this commit?

Reply:
Good catch. Done now; just merged this file in in the last PR so now it's pointing to the right place.